docs/Building.md

```diff
@@ -116,6 +118,35 @@ and llvm.
     source utils/env_setup.sh <mlir-aie>/install <llvm dir>/install
     ```
 
+## Building on X86 targetting the VCK5000
+
+In order to build and run on PCIe cards, you first have to build and install the aie-rt library. We chose to install the library in /opt/xaiengine but it is not required for the tools to be installed there. Just ensure that when building mlir-aie and mlir-air, that you point to the directory in which the aie-rt library was installed.
+
+```
+git clone https://github.com/stephenneuendorffer/aie-rt
+cd aie-rt
+git checkout phoenix_v2023.2
+cd driver/src
+make -f Makefile.Linux CFLAGS="-D__AIEAMDAIR__"
+sudo cp -r ../include /opt/aiengine/
+sudo cp libxaiengine.so* /opt/xaiengine/lib/
+export LD_LIBRARY_PATH=/opt/xaiengine/lib:${LD_LIBRARY_PATH}
+```
```

Comment:
Do we really need this or can we use the aie-rt submodule?

Reply:
Decided to do this in another PR.